tokens.py

```diff
@@ -0,0 +1,30 @@
+import hashlib
+from binascii import hexlify, unhexlify
+from collections import namedtuple
+import secrets
+
+token_len = 32
```

Comment:
I guess token length should be used at line 32 and 28.

Reply:
True, I forgot to use that variable, I should probably rename and use it.